packages/grafana-data/src/types/icon.ts

```diff
@@ -4,6 +4,11 @@ export const availableIconsIndex = {
   github: true,
   gitlab: true,
   okta: true,
+  discord: true,
```

Comment:
why do we need this in this PR?

Reply:
You can load any icon from [unicons](https://iconscout.com/unicons/explore/line) but adding them here adds code completion to vscode.

Reply:
Oh and because Discord is a supported contact point and we show an icon for it when we can :)